katas/content/superposition/greenberger_horne_zeilinger/solution.md

```diff
@@ -0,0 +1,26 @@
+### Solution
+
+The single-qubit GHZ state is the plus state $\frac{1}{\sqrt{2}} \big (|0\rangle + |1\rangle\big)$ that we've discussed in task 1.1. As a reminder, that state is prepared by applying a Hadamard gate.
+
+The 2-qubit GHZ state is the Bell state $\frac{1}{\sqrt{2}} \big (|00\rangle + |11\rangle\big)$ that we've discussed in task 1.6.
+
+The next one is the 3-qubit GHZ state:
+$$|GHZ\rangle = \frac{1}{\sqrt{2}} \big (|000\rangle + |111\rangle\big)$$
+
+Let's use the 2-qubit circuit as a building block to construct the circuit for 3 qubits. First, let's add a third qubit to the above circuit.
+
+Comparing the state prepared by this circuit with the desired end state, we see that they differ only in the third (rightmost) qubit:
```

Comment:
We need to rephrase this to account for not having circuits and talking about states instead. How about
```suggestion
Let's use the 2-qubit state as a building block to construct the state for 3 qubits. First, let's add a third qubit to the above state (on the right from the first two qubits).
Comparing this state with the desired end state, we see that they differ only in the third (rightmost) qubit:
```